README.md

```diff
@@ -154,15 +190,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+Person.create (id: 1, name: "Sandra", age: 44)
```

Comment:
You should never hardcode a primary key when creating something. Just `name` and `age` will do here.